lang/rust/avro/src/schema.rs

```diff
@@ -3305,16 +3314,16 @@ mod tests {
 
         let schema = Schema::parse_str(raw_schema)?;
         assert_eq!(
-            "abf662f831715ff78f88545a05a9262af75d6406b54e1a8a174ff1d2b75affc4",
+            "b18ddbf029afaa876b54cb4de997506ef67e202bee13d8b54cb5698288c25318",
```

Comment:
I think this is a regression.
According to https://avro.apache.org/docs/1.11.1/specification/#transforming-into-parsing-canonical-form we should `STRIP` everything but `type, name, fields, symbols, items, values, size`. I.e. the custom attributes should be stripped and the fingerprint should stay the same.